### DISCUSSION TITLE
Our branding is replaced by Wiki.js white flower brand

### DISCUSSION THREAD

**Discussion by darashayda** [requarks/wiki#6199] [Error / Bug Report]
Same is happening, as I reported in all the messaging software on Mac OS and IOS.

<img width="1415" alt="Screen Shot 2023-02-22 at 6 30 05 PM" src="https://user-images.githubusercontent.com/5667455/220723843-077a7613-04f5-4204-bf97-5ddc6392ddd2.png">

**Comment by Philipinho** [comment]
Are you talking about the Favicon at the title bar?

**Reply by darashayda** [reply]
I guess the WHITE FLOWER Wiki.js branding logo, as you can see WHITE FLOWER renders where our Brand logo C^C^N was supposed to appear. 

Please review the attached images in the first post

**Reply by Philipinho** [reply]
It is called favicon. If you use docker, you can bind mount the file `/wiki/assets/favicon.ico` to override it with your own favicon (logo).

**Reply by darashayda** [reply]
No I do not use docker, unless something I could pick up quick we are at release time (behind actually) so time is of the essence. 

Let us know if there is another way from Wiki dashboard?

**Reply by Philipinho** [reply]
You can replace this file with your own favicon.ico -> [/wiki/client/static/favicon.ico](https://github.com/requarks/wiki/blob/3a0ec8d7fcd9c7b74591394ffa1908088e635770/client/static/favicon.ico) locally. Note that updates will override any changes. You will have to replace it again during updates.

**Reply by darashayda** [reply]
Thank you, I cannot come back to this again, so I need a better way. Not want to impose on your time, but either let me know how to use this docker thingy or some other way please. 

Much obliged to thank you for this generous gesture

D

**Reply by Philipinho** [reply]
If you use Nginx as a reverse proxy on top WIki.js, you can use the location directive to rewrite the favicon.
```
location = /favicon.ico {
        # The path to your custom favicon.ico file
        alias /path/to/custom/favicon.ico;
}
```

If you use Apache:
`Redirect 301 /favicon.ico /path/to/custom/favicon.ico`

**Reply by darashayda** [reply]
Thank you, I need to ask my IT fellows to do these, I know you game exact and proper answers. 

Will apply these and report back. 

THank you

**Reply by fnafox** [reply]
Replacing just favicon.ico is not enough, you also need to replace all instances of the site icon in the `wiki/assets/favicons` directory as iOS and Saferi use apple-touch-icon.png and safari-pinned-tab.svg respectively. There are online tools that will create all necessary files for you from your provided image.

**Reply by Philipinho** [reply]
Yea, I realized that too after checking the `favicons` directory and the rendered wiki.js page. 
More lines of rewrite rule to cater for them all.

**Reply by darashayda** [reply]
Much  obliged to express my gratitude to all of you. 

Our expertise are so far removed from attempting to reach your technical expertise. 

There seems to be loose-ends around some of these issues, which of course is expected, and no criticism for the developers, but we have to rely on the community to address them. We just do not have the bandwidth to go to this extend. 

Also since  I have your attentions, could someone send me the Docs for STICKY Table Of Contents or FLOATING TOC?

I swear when I signed up the Headers appeared in a floating TOC and suddenly disappeared! Not sure.
